tests/test_vehicle_info_spec.py

```diff
@@ -0,0 +1,275 @@
+# SPDX-License-Identifier: MIT
+import inspect
+from pathlib import Path
+from typing import Any
+from xml.etree import ElementTree
+
+import jinja2
+
+import odxtools
+from examples.somersaultecu import database as somersault_db
+from odxtools.ecuproxy import EcuProxy
+from odxtools.gatewaylogicallink import GatewayLogicalLink
+from odxtools.memberlogicallink import MemberLogicalLink
+from odxtools.nameditemlist import NamedItemList
+from odxtools.odxlink import DocType, OdxDocFragment
+from odxtools.oem import Oem
+from odxtools.writepdxfile import (jinja2_odxraise_helper, make_bool_xml_attrib, make_ref_attribs,
+                                   make_xml_attrib, set_category_docfrag, set_layer_docfrag)
+
+doc_frags = (OdxDocFragment(doc_name="vehicle_info_test", doc_type=DocType.VEHICLE_INFO_SPEC),)
+
+vehicle_info_spec_xml_str = """<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
Better to move it to its own file

Reply:
I'd like to keep the unit tests as self contained as possible. The tests for flashing, variant coding and multiple ecu jobs follow the same pattern..